components/Layout/Images/BlurhashImage.vue

```diff
@@ -58,6 +64,11 @@ export default Vue.extend({
     alt: {
       type: String,
       default: ''
+    },
+    iconSize: {
+      type: String,
+      required: false,
+      default: ''
```

Comment:
Should this be an actual size instead of `''`

Reply:
Or if `''` is an acceptable input why not `undefined`

Reply:
Oh good catch. I wanted before to accept numbers only but the size prop of v-icon actually accepts strings and numbers (so values like 7em are possible). I'm going to set the 7em value we use by default in the computed property here 👍.